packages/mdc-slider/index.js

```diff
@@ -162,6 +165,44 @@ class MDCSlider extends MDCComponent {
     this.max = parseFloat(this.thumb_.getAttribute(strings.ARIA_VALUEMAX)) || this.max;
     this.step = parseFloat(this.thumb_.getAttribute(strings.DATA_STEP)) || this.step;
     this.value = origValueNow || this.value;
+    if (this.tickMarkSet_) {
+      this.setUpTickMarks();
+    }
+  }
+
+  setUpTickMarks() {
+    if (this.step < 1) {
+      this.step = 1;
+    }
+    let numMarks = (this.max - this.min) / this.step;
+
+    // In case distance between max & min is indivisible to step,
+    // we place the secondary to last mark proportionally at where thumb
+    // could reach and place the last mark at max value
+    const indivisible = Math.ceil(numMarks) !== numMarks;
+    if (indivisible) {
```

Comment:
You can remove the `indivisible` check, since nothing would happen if you call `Math.ceil` when it's false

Reply:
got it!